skycultures/bali/index.json

```diff
@@ -0,0 +1,859 @@
+{
+  "id": "bali",
+  "region": "South-eastern Asia",
+  "classification": ["Ethnographic"],
+  "fallback_to_international_names": false,
+  "constellations": [
+      {
+      "id": "CON bali 001",
```

Comment:
The numerical IDs are ugly. In display mode "abbreviations" you would have a chance to display short unique strings (2-4 Latin characters, like 'KaSu' or 'KSu') as mnemonic of the name. This helps people familiar with the skyculture who don't want to clutter their display. Placing serial numbers as labels is IMO not very intuitive.

Reply:
oh, great idea - the numbers in this case *do* have a meaning (like numbering zodiac signs or nakshatras), as they are in the historical source that was used - but you're right, we should make this clear by naming them something like "lintang xy".